### PR TITLE
Add SPDX List v3.16 to LicenseListVersion and the file format changelog

### PR DESCRIPTION
**Pull request by Kleidukos**
Follow-up of #8127

### CODE REVIEW

**Comment by Mikolaj** [comment]
@Kleidukos: what do you think about what @fgaz's mentioned?

**Comment by Kleidukos** [comment]
Yes that's excellent!
I've seen the stuff that's missing, I'll add them today :)

**Comment by Kleidukos** [comment]
@Mikolaj @fgaz Updated!

File: Cabal-syntax/src/Distribution/SPDX/LicenseId.hs
Comment (Mikolaj) [inline Cabal-syntax/src/Distribution/SPDX/LicenseId.hs:1867]: Wow, so the list for 3.10 was buggy?

**Review by Mikolaj — APPROVED** [review]
Thank you. Once this merges I will cut 3.8.

File: Cabal-syntax/src/Distribution/SPDX/LicenseId.hs
Comment (Kleidukos) [reply Cabal-syntax/src/Distribution/SPDX/LicenseId.hs:1867]: yup' :(

**Comment by Mikolaj** [comment]
Actually, I will cut 3.8 without waiting for the merge. I'd like to report problems on tomorrow's meeting, if any. Let's remember to backport this to 3.8 once it merges.

**Review by fgaz — APPROVED** [review]
Thanks!

**Comment by fgaz** [comment]
@Mergifyio backport 3.8

**Comment by fgaz** [comment]
Oh oops we didn't branch-off yet

**Comment by Mikolaj** [comment]
We did now. With this PR included for free. :)

**Pull request merged**
